assets/src/scripts/store/network.js

```diff
@@ -0,0 +1,71 @@
+import {
+    fetchObservationItem
+} from '../web-services/observations';
+
+const INITIAL_DATA = {
+    networkList: []
+};
+
+/*
+ * function to build a network URL from a labs json url
+ * @param {String} link
+ * @return {String} network link
+ */
+export const buildNetworkURL = function(link) {
```

Comment:
This seems like it is the wrong place to do this work. Our pattern has been to store the data that we fetch in the Redux store and then let selectors do the work of getting the data that we need in the view code.  you could just build the links as you render them in components/network.js and move this bit of logic there. Or you could add a selector in the components/network to return the list of url's and use that to render the list.

Reply:
I moved the buildNetworkURL function in to the network component.